ds-prep-python-lists/main.py

```diff
@@ -0,0 +1,21 @@
+ages = [29 , 19 , 17 , 26 , 32 , 16 , 21]
+print(ages)
+print(ages[4])
+print(ages[-2])
+print(ages[2:6])
+print(ages[:4])
+print("17" in ages)
+print("42" in ages)
+ages[2] = "18"
+print(ages)
+ages[4] = "temp."
+ages [4] = ages[3]
+ages[3] = 32
+print(ages)
```

Comment:
- `"temp."` is a string, not a variable name.
- The fifth value in `ages` should be assigned to `temp`, not vice versa.
- The fourth value in `ages` should be overwritten with the value in the `temp` variable, not the hard-coded number `32`.

---

![image](https://user-images.githubusercontent.com/16143771/149632188-97ff1580-efc4-43ec-b11b-3a17a765c65d.png)